credential-trust-establishment/spec.md

```diff
@@ -18,118 +18,257 @@ Participate:
 ------------------------------------
 
 ## Introduction
-
-Once trust lists are established, it is natural to extend trust establishment by defining roles and linking them to credentials for situations where
+Once you can create a trust list using Trust Establishment, it is useful to extend Trust Establishment by defining roles and linking them to credentials for situations where:
```

Comment:
This is fine, but it would be great to have 1. Trust List more clearly defined in PE and 2. Link to the PE documentation here as an embedded link.